### PR TITLE
Port to go.mod

### PR DESCRIPTION
**Pull request by WillNilges**

### CODE REVIEW

**Comment by WillNilges** [comment]
Lol lemme add those guys as well

File: env-template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export S3_SECRET_KEY=
+export csh_auth_jwt_secret=
+export csh_auth_server_host=
+export csh_auth_redirect_uri=
+export DB_URI=
+export LDAP_BIND_PW=
+export LDAP_BIND_DN=
+export LDAP_HOST=
+export GIN_MODE=release
+export BASE_PATH=
```
Comment (mxmeinhold) [inline env-template:15]: Some of these few like they need some contextualisation

**Review by mxmeinhold — COMMENTED** [review]
You committed a binary?

What's the value of the run env script? I feel like that's something that should at least be contextualised in the readme.

Is vendoring necessary? I feel like the checksums file deals with most of what we would need vendoring for, but maybe I'm missing something.

**Review by Mstrodl — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM golang:1.16 as builder
+
+RUN mkdir /build
+COPY * /build
+WORKDIR /build
```
[the remark this answers is not in the document]
Comment (WillNilges) [reply Dockerfile:5]: ur so right

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# RUN go fmt github.com/computersciencehouse/csh-plug # TODO (willnilges): Should I do this?
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o csh-plug .
+#CMD ["/go/csh-plug"] # Bonk
```
[the remark this answers is not in the document]
Comment (WillNilges) [reply Dockerfile:8]: Nah

File: env-template (hunk 1 above)
Comment (WillNilges) [reply env-template:15]: I can document these

**Pull request merged**